src/content/docs/en/welcome.mdx

```diff
@@ -19,11 +19,11 @@ To do this, we stick to a set of values:
 
 ### Ultramarine is Pragmatic.
 
-Where Fedora tries to adhere to ideology (that’s perfectly fine), by shipping an (almost) entirely free system, Ultramarine works off the model of least resistance. That means we try to make it easy for the user, even if we have to ship some proprietary drivers and codecs that Fedora wouldn’t be comfy with out-of-box.
+While Fedora ships a primarily free system by default, Ultramarine works off the model of least resistance. We try to make it easy for the user, even if we have to ship some proprietary drivers, firmware, and codecs that Fedora wouldn’t be comfy with out-of-box.
 
 ### Ultramarine is Innovative.
 
-We believe that recent innovations in the Linux space serve to greatly improve the user and developer experience. Technologies like Flatpak and xdg-portals allow developers to provide great experiences to our users more consistently. Work done upstream improves the experience for our users, and work done in the Ultramarine ecosystem; like accent colours, and Chromebook support, improves the larger Linux ecosystem. It's a win-win.
+We believe that recent innovations in the Linux space serve to greatly improve the user and developer experience. Technologies like Flatpak and xdg-portals allow developers to provide great experiences to our users more consistently. Work done upstream and in Ultramarine improves the experience for our users, providing features like accent colours and Chromebook support that improve the larger Linux ecosystem.

```

Comment:
no to this one, we're trying to demonstrate the circular flow of improvements coming from upstream and our work flowing back, this change removes that